README.md

```diff
@@ -10,24 +10,47 @@ A JavaScript plugin to warn users about links to private pages. Places a :lock:
 
 At 18F, this is used on public sites that contain links to internal content like private GitHub repositories or Google Docs. Rather than write two versions to redact those links, this allows us to publish new content and give a warning to both staff and external readers.
 
-## Usage
+## Installation
 
 Compatible with modern browsers (IE 9+). No dependencies.
 
+### Script
+
+Private Eye can be included as a normal script on your page, exposing a `PrivateEye` global.
+
 ```html
 <script src="private-eye.js"></script>
```

Comment:
Might be nice to use the `defer` attribute on the script tag. Browser support is pretty good and since you're using `DOMContentLoaded` it's safe to assume the script has been executed. Not a big deal, but since the use case might be simply including the script tag in the `README` along with the usage snippet ( lightly edited ), it's a simple performance gain. https://webkit.org/blog/1395/running-scripts-in-webkit/